docker build failed due to gradle oneJar in job-dsl-plugin

The error log while running command

```
docker build -t jdsl .
```
Log

```
Download https://plugins.gradle.org/m2/org/jenkins-ci/annotation-indexer/1.4/annotation-indexer-1.4.jar

FAILURE: Build failed with an exception.

* Where:
Build file '/jdsl/job-dsl-plugin/build.gradle' line: 14

* What went wrong:
An exception occurred applying plugin request [id: 'org.jenkins-ci.jpi', version: '0.38.0']
> Failed to apply plugin [id 'org.jenkins-ci.jpi']
   > Could not find method register() for arguments [server, class org.jenkinsci.gradle.plugins.jpi.ServerTask, org.jenkinsci.gradle.plugins.jpi.JpiPlugin$_apply_closure1@697889b6] on task set of type org.gradle.api.internal.tasks.DefaultTaskContainer.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 34s
The command '/bin/sh -c true  && git clone https://github.com/jenkinsci/job-dsl-plugin.git /jdsl  && cd /jdsl  && gradle oneJar' returned a non-zero code: 1
```
Seems job-dsl-plugin moves to gradle 5.6.4 version.

--- a/Dockerfile
+++ b/Dockerfile
@@ -22,19 +22,19 @@ RUN true \
  && apt-get update \
  && apt-get install -qy --no-install-recommends openjdk-8-jdk openjdk-8-jre \
  && apt-get clean \
- && rm -rf /var/lib/apt/lists/* 
+ && rm -rf /var/lib/apt/lists/*
 
 RUN true \
  && apt-get update \
  && apt-get install -qy --no-install-recommends curl\
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
- 
-RUN curl -sL https://services.gradle.org/distributions/gradle-4.6-bin.zip -o gradle-4.6.zip \
- && unzip -d /usr/share gradle-4.6.zip \
- && ln -s /usr/share/gradle-4.6/bin/gradle /usr/bin/gradle \
+
+RUN curl -sL https://services.gradle.org/distributions/gradle-5.6.4-bin.zip -o gradle-5.6.4.zip \
+ && unzip -d /usr/share gradle-5.6.4.zip \
+ && ln -s /usr/share/gradle-5.6.4/bin/gradle /usr/bin/gradle \
  && gradle --version \
- && rm gradle-4.6.zip
+ && rm gradle-5.6.4.zip
 
 RUN true \
  && apt-get update \
@@ -46,7 +46,7 @@ RUN true \
 RUN true \
  && git clone https://github.com/jenkinsci/job-dsl-plugin.git /jdsl \
  && cd /jdsl \
- && gradle :job-dsl-core:oneJar
+ && gradle oneJar
 
 ENV DSL_JAR "/jdsl/job-dsl-core/build/libs/job-dsl-core-*-standalone.jar"
 

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ Inside the container, run:
 
 Start the container:
 
-    docker run -d --name jdsl -p 3000:3000 jdsl
+    docker run -d --rm --name jdsl -p 3000:3000 jdsl
 
 Upload a job for translation:
 

